policyengine_us/reforms/biden/budget/medicare_and_investment_tax_increase.py

```diff
@@ -0,0 +1,80 @@
+from policyengine_us.model_api import *
+
+
+def create_medicare_and_investment_tax_increase() -> Reform:
+    class additional_medicare_tax(Variable):
+        value_type = float
+        entity = TaxUnit
+        definition_period = YEAR
+        label = "Additional Medicare Tax"
+        unit = USD
+        documentation = (
+            "Additional Medicare Tax from Form 8959 (included in payrolltax)"
+        )
+
+        def formula(tax_unit, period, parameters):
+            amc = parameters(period).gov.irs.payroll.medicare.additional
+            # Wage and self-employment income are taxed the same.
+            ELEMENTS = [
+                "irs_employment_income",
+                "taxable_self_employment_income",
+            ]
+            wages_plus_se = add(tax_unit, period, ELEMENTS)
+            exclusion = amc.exclusion[tax_unit("filing_status", period)]
+            base = max_(0, wages_plus_se - exclusion)
+            base_tax = amc.rate * base
+            p_ref = parameters(period).gov.contrib.biden.budget.medicare
```

Comment:
I thought `ref` = `reference`
```suggestion
            p_reform = parameters(period).gov.contrib.biden.budget.medicare
```